framework/Models/PostProcessors/BasicStatistics.py

```diff
@@ -999,20 +999,21 @@ def __runLocal(self, inputData):
         targDa = dataSet[target]
         if self.pivotParameter in targDa.sizes.keys():
           percentileSte = []
-          for pct in percent:
+          for label, group in targDa.groupby(self.pivotParameter):
```

Comment:
These changes will decrease the running time for the following test from 480s to 335s. 
![image](https://user-images.githubusercontent.com/7321071/155018916-a8dad750-f5dc-4352-a2e2-813cf857cca1.png)

The original test before the add of error calculation for percentile is about 130s.

Reply:
Have we run any tests on the accuracy of the "standard normal" assumption versus this fitted function? I'm sure it's heavily reliant on the particular model being sampled, but some idea of the error introduced would inform how valuable the more accurate calculation is.